compile.sh

```diff
@@ -37,7 +36,25 @@ if [ "$_has_dtbo" = "true" ]; then
 fi
 
 _self_dir=$(realpath $(dirname "$0"))
-. $_self_dir/compile_$_compiler.sh
+# TODO: Using source here so that all environment vars are inherited
+. $_self_dir/setup_$_compiler.sh
+_cross_compile_32="$ANDROID_ROOT/prebuilts/gcc/linux-x86/arm/arm-linux-androideabi-4.9/bin/arm-linux-androideabi-"
+_build_cmd="make O=$_out ARCH=arm64 -j$(nproc) $_make_args CROSS_COMPILE_ARM32=$_cross_compile_32"
+
+_defconfig=aosp_${_platform}_${_device}_defconfig
+
+echo "==> Entering $_kernel_path"
+pushd "$_kernel_path" || (echo "ERROR: Failed to cd into kernel source!"; exit 1)
+
+    echo "==> Building $_defconfig"
+    $_build_cmd $_defconfig
+
+    echo "==> Building $_targets with $_compiler"
+    time $_build_cmd $_targets
```

Comment:
I like the `time` command there, but it was only before in the linaro case. Was it intended to stay there or it was a copy-paste?

Reply:
Yeah that's intended so that one can see how long the compilation took :)